tools/model-compiler/src/main/java/io/spine/tools/compiler/DirectoryCleaner.java

```diff
@@ -43,8 +43,10 @@
     public static void deleteDirs(List<String> dirs) {
         for (String dirPath : dirs) {
             File file = new File(dirPath);
-            if (file.exists() && file.isDirectory()) {
-                deleteRecursively(file.toPath());
+            if (file.isDirectory()) {
+                if (file.exists()) {
```

Comment:
`File.isDirectory()` returns `false` if the file does not exist. Thus, this check is redundant.